app/models/shipit/commit.rb

```diff
@@ -143,7 +143,7 @@ def schedule_refresh_check_runs!
 
     def refresh_statuses!
       github_statuses = stack.handle_github_redirections do
-        Shipit.github.api.statuses(github_repo_name, sha, per_page: 100)
+        stack.repository.github_api.statuses(github_repo_name, sha, per_page: 100)
```

Comment:
`Stack` should delegate `github_api` to it's `repository` to avoid these long chains everywhere.

Reply:
👍